src/desktop/osx/CMakeLists.txt

```diff
@@ -1,37 +1,19 @@
 # Desktop Integration
-# Copyright (c) 2022  Igara Studio S.A.
+# Copyright (c) 2022-2025  Igara Studio S.A.
 
-find_library(QUARTZ_LIBRARY Quartz)
+find_library(QUICKLOOK_LIBRARY QuickLookThumbnailing)
 
-add_library(AsepriteThumbnailer SHARED
-  main.mm
+set(extension_target AsepriteThumbnailer)
+
+add_executable(${extension_target} MACOSX_BUNDLE
```

Comment:
Could we just output the `.appex` inside the `bin/Aseprite.app/Contents/PlugIns` directory? Not sure if it does make sense (or if it simplify the build scripts later).